rust/src/client/configuration/telemetry_configuration.rs

```diff
@@ -62,14 +63,13 @@ impl TryFrom<&Properties> for TelemetryConfiguration {
     fn try_from(properties: &Properties) -> Result<Self, Self::Error> {
         let section = ("telemetry", properties);
 
+        // optionals
         let path = match get_optional_from_section::<String>("path", properties) {
             Ok(value) => value.unwrap_or(DEFAULT_PATH.to_string()),
             Err(e) => {
                 warn!(
-                    "OLTP collector path could not be read from configuration: {}",
-                    e
+                    "OLTP collector path cannot be read from configuration: {e}, defaulting to '{DEFAULT_PATH}'"
```

Comment:
Better put the error at the end than in the middle of the log, or force a new line (but it would be the same as having two logs actually)

`Defaulting OLTP collector path to '{DEFAULT_PATH}' as it cannot be read from configuration: {e},`